rotkehlchen/db/upgrades/v24_v25.py

```diff
@@ -469,7 +469,25 @@ def upgrade_v24_to_v25(db: 'DBHandler') -> None:
     cursor.execute(
         f'DELETE FROM used_query_ranges WHERE name LIKE "{BALANCER_EVENTS_PREFIX}%";',
     )
-    cursor.execute('DELETE FROM uniswap_events;')
+    cursor.execute('DROP TABLE IF EXISTS uniswap_events;')
```

Comment:
Everything should be ok now. There was an issue with this upgrade when coming from an older version (in the test for restoring the premium synced db), seems that the table was created using the initialization in the db but is not the case anymore since i deleted that sentence from the `schema.py` file.

Reply:
As I wrote above, don't do the upgrade now.